server/webapp/WEB-INF/rails.new/app/presenters/api_v2/config/materials/svn_material_representer.rb

```diff
@@ -18,17 +18,22 @@ module ApiV2
   module Config
     module Materials
       class SvnMaterialRepresenter < ScmMaterialRepresenter
-        property :check_externals, exec_context: :decorator
+        property :check_externals
         property :user_name, as: :username
         property :password,
                  skip_render: true,
-                 skip_nil: true,
-                 setter:      lambda { |value, options|
-                   self.setCleartextPassword(value)
-                 }
+                 skip_nil: true
         property :encrypted_password, skip_nil: true
 
-        delegate :check_externals, :check_externals=, to: :material_config
+        def from_hash(data, options={})
+          super
+          data = data.with_indifferent_access
+          represented.setCheckExternals(data[:check_externals].to_bool)
+          represented.setUserName(data[:username])
+          encrypted_password = PasswordDeserializer.new.deserialize(data[:password], data[:encrypted_password], represented)
+          represented.setEncryptedPassword(encrypted_password)
+          represented
```

Comment:
This can actually be a bit simpler —

```ruby
property :password, skip_parse: true, ...
property :encrypted_password, skip_parse: true, ...

def from_hash(data, options={})
  super
  data = data.with_indifferent_access
  encrypted_password = PasswordDeserializer.new.deserialize(data[:password], data[:encrypted_password], represented)
  represented.setEncryptedPassword(encrypted_password)
  represented
end

```